spec/requests/swagger_docs/v4/capitals_spec.rb

```diff
@@ -104,7 +104,7 @@
 
         run_test! do |response|
           body = JSON.parse(response.body, symbolize_names: true)
-          expect(body[:errors]).to include(/Missing parameter value/)
+          expect(body[:errors]).to include("ActiveRecord::RecordInvalid: Validation failed: Value can't be blank")
```

Comment:
It looks to me like the json-schema validation isn't working as expected if you are getting the ActiveRecord::RecordInvalid error. (To me this implies we are either not calling json_validator before trying to save the record or the validation is not correct).